### ISSUE TITLE
Feature request: Creating new github repositories

### ISSUE THREAD

**Issue by imhuay** [KnisterPeter/vscode-github#111]
Excuse, I am first to use Git remotely
And The first step `GitHub: Set Personal Access Token` just fails
```
Command failed: git config --local --get remote.origin.url
```
How can solve it, thanks

**Comment by KnisterPeter** [comment]
I think your workspace (if you have opened one) has no valid .git folder.
Can you check that?

**Comment by imhuay** [comment]
@KnisterPeter There is a .git folder.
Does the `vscode-github` need to config the remote github url on my own?

**Comment by KnisterPeter** [comment]
Whats the output if you execute `git config --local --get remote.origin.url` in a terminal in your workspace folder?

**Comment by KnisterPeter** [comment]
Do you have the git executable in your PATH? What OS are you using?

**Comment by imhuay** [comment]
@KnisterPeter Win10
I can use git local.
There is no output execute `git config --local --get remote.origin.url`

When I use VSCode to release, it tell me 
`
Warning: Your repository has no remotes configured to publish to.
`

**Comment by KnisterPeter** [comment]
The output on the terminal should be something like this:

```shell
$ git config --local --get remote.origin.url
git@github.com:KnisterPeter/vscode-github.git
```

If possible can you please paste the content of the '.git/config'  file from your project folder?

**Comment by imhuay** [comment]
`.git/config`
```
[remote "origin"]
    url = https://github.com/imhuay/xxx.git
```
I add it manually, then `vscode-github` works.
I think if `vscode-github` can help me do this will be better

**Comment by KnisterPeter** [comment]
@imhuay I still does not get it. How did you clone your project without git having to create your .git/config remote? Or was it a new project at all?

**Comment by Defake** [comment]
Also having this issue. Created new project, and got this message on try to add token. 
Is this impossible to easily create new repositories on github right from IDE like in Visual Studio 2015?

**Comment by KnisterPeter** [comment]
@Defake How exactly did you create your project? If you create an empty repo on github and then clone it it should work as expected.
The above error results from not having an `origin` remote added to your local git repository. Is that possible?

**Comment by Defake** [comment]
@KnisterPeter Yes, I successfully performed this actions: initialized new local repository (from vscode), created repository on github.com, added remote "origin", pushed local master to origin. 

That works fine but I remember that in Visual Studio 2015 it was possible to create local repository and then create github repository with just one click. After trying to create github repo like this I got that issue:
`Command failed: git config --local --get remote.origin.url`

**Comment by KnisterPeter** [comment]
I'll change this to a feature request for creating repositories with a command.
Nice idea.

**Comment by notxcain** [comment]
I'm getting strange error `Command failed: git config --local --get remote..url`, like it doesn't see any remote configured.

**Comment by KnisterPeter** [comment]
This is a regression fixed by #150 
I'll create a release in a few moments

**Comment by notxcain** [comment]
@KnisterPiter awesome, thank you
